hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkUncheckedSubmitHandler.java

```diff
@@ -55,4 +59,14 @@ public void handle(@NonNull final HandleContext context) throws HandleException
         requireNonNull(context);
         throw new HandleException(NOT_SUPPORTED);
     }
+
+    @NonNull
+    @Override
+    public Fees calculateFees(@NonNull final FeeContext feeContext) {
+        requireNonNull(feeContext);
+
+        return feeContext
+                .feeCalculator(SubType.DEFAULT)
+                .legacyCalculate(sigValueObj -> UncheckedSubmitResourceUsage.usageGiven());
```

Comment:
We don't need this here, if it is just default Fees

Reply:
See my reply above in UncheckedSubmitResourceUsage